stage2/litellm/README.md

```
@@ -0,0 +1,163 @@
+  ```
+
+  Longhorn supports online expansion, so no downtime is needed. Shrinking is not possible.
+- **Rotating `litellm_db_password` breaks the deployment.** The postgres image only reads
+  `POSTGRES_PASSWORD` when the data directory is empty, so a new value updates the Secret and the
+  clients but not the database role, and every connection starts failing authentication. Change the
+  role first, then the variable:
+
+  ```bash
+  kubectl -n litellm exec -it litellm-postgres-0 -- \
+    psql -U litellm -d litellm -c "ALTER USER litellm WITH PASSWORD '<new>';"
+  ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)